base/exports.jl

```diff
@@ -1009,6 +1009,8 @@ export
 # dates
     Date,
     DateTime,
+    DateFormat,
+    @dateformat_str,
```

Comment:
If you meant I need to add this to the manual, I have. I was wondering if there's some kind of list I need to update when you said "manual index"